mobile/native/src/ln_dlc/mod.rs

```diff
@@ -249,7 +249,7 @@ pub fn run(data_dir: String, seed_dir: String, runtime: &Runtime) -> Result<()>
     })
 }
 
-async fn keep_wallet_balance_and_history_up_to_date(node: &Node) -> Result<()> {
```

Comment:
🤔 Haha, caught you! This belongs in the previous patch!

Reply:
damn!